templates/ray-data-batch-classification/README.ipynb

```diff
@@ -24,10 +24,22 @@
     "\n",
     "See [this guide on batch inference](https://docs.ray.io/en/latest/data/batch_inference.html#batch-inference-home) for tips and troubleshooting when adapting this example to use your own model and dataset!\n",
     "\n",
-    "To run this example, you will need the following packages:\n",
-    "\n",
-    "```!pip install -q \"ray[data]\" torch torchvision```\n",
-    "\n",
+    "To run this example, you will need the following packages:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!pip install -q \"ray[data]\" torch torchvision"
```

Comment:
are the `\` here auto-generated from running the pre-commit? if they are not needed / not auto-generated, can we remove them?

Reply:
yeah it is auto generating these changes 🤔 I removed it but it still brought it back